### ISSUE TITLE
Unable to signoff commits since v4

### ISSUE THREAD

**Issue by geemanjs** [google-github-actions/release-please-action#904]
### TL;DR

As per the title - It appears as if though the interface between release-please-action and release-please itself results in the `signoff` property being ignored. 

### Expected behavior

When migrating from v3 -> v4 
Add the `signoff` key to my `release-please-config.json` file 
Release please continues to signoff commits

### Observed behavior

No commit signing appears to be happening 

### Action YAML

```yaml
Happy to share this but it's mostly irrelevant. Heres the original release-please config:

      - name: Release please
        uses: google-github-actions/release-please-action@v4
        id: release
        with:
          token: ${{ secrets.RP_TOKEN}}
          command: manifest
          monorepo-tags: true
          signoff: "Obfuscated <obfuscated@obfuscated.com>"


And heres the two keys I added to the `release-please-config.json`
```json
  "signoff": "Obfuscated <obfuscated@obfuscated.com>",
  "include-component-in-tag": true,
```
```


### Log output

_No response_

### Additional information

Having done some digging.. 

The `Manifest` constructor expects the `signoff` property to be in `manifestOptions` (the 5th argument to the constructor)
https://github.com/googleapis/release-please/blob/main/src/manifest.ts#L356 

When `fromManifest` is called it sets the 5th argument of the Manifest constructor to [the following](https://github.com/googleapis/release-please/blob/main/src/manifest.ts#L409-L427):
```json
{
  manifestPath: manifestFile,
  ...manifestOptions,
  ...manifestOptionOverrides,
}
```
`manifestOptionOverrides` is only passed in by [release-please-action in certain circumstances](https://github.com/google-github-actions/release-please-action/blob/main/src/index.ts#L104-L110)
`manifestOptions` comes from the `parseConfig` function in release-please

The `parseConfig` function does not extract the `signoff` property into the `options`:
https://github.com/googleapis/release-please/blob/main/src/manifest.ts#L1392-L1406

**Comment by jensdrenhaus** [comment]
According to the linked issue and PR, the bug is fixed in recent release-please version. 
Running my workflow with release-please-action@v4 it prints in the logs `Running release-please version: 16.12.0`

However, the commit in the release-pr is still not signed of in my repo. 

Action YAML
```
name: Release

on:
  push:
    branches:
      - main
  workflow_dispatch:

permissions:
  contents: write
  pull-requests: write

jobs:
  release-please:
    name: Maintain Upcoming Release
    runs-on: ubuntu-latest
    steps:
      - uses: googleapis/release-please-action@v4
        id: release
        with:
          token: ${{ secrets.RELEASE_PLEASE_TOKEN }}
          config-file: release-please-config.json
          manifest-file: .release-please-manifest.json
```

release-please-config.json:
```
{
    "release-type": "go",
    "bump-minor-pre-major": true,
    "pull-request-title-pattern": "chore: release ${version}",
    "include-component-in-tag": false,
    "include-v-in-tag": true,
    "signoff": "Jens Drenhaus <jens.drenhaus@blindspot.software>",
    "changelog-sections": [
        {
            "type": "feat",
            "section": "Features"
        },
        {
            "type": "fix",
            "section": "Bug Fixes"
        },
        {
            "type": "docs",
            "section": "Documentation"
        },
        {
            "type": "refactor",
            "section": "Other Work"
        },
        {
            "type": "test",
            "section": "Other Work"
        },
        {
            "type": "build",
            "section": "Other Work"
        },
        {
            "type": "ci",
            "section": "CI",
            "hidden": true
        },
        {
            "type": "chore",
            "section": "Chore",
            "hidden": true
        },
        {
            "type": "revert",
            "section": "Revert",
            "hidden": true
        }
    ],
    "packages": {
        ".": {}
    }
}
```

**Comment by jensdrenhaus** [comment]
@geemanjs do you still encounter the problem, too?

**Comment by shanduur** [comment]
That's still an issue - any options to solve it?
